web/views.py

```
@@ -158,6 +158,10 @@
     UserBadge,
     WaitingRoom,
     WebRequest,
+    CounterStatistic,
```

Ruff (0.8.2)

`.models.CounterStatistic` imported but unused

Remove unused import

(F401)